corehq/apps/app_manager/models.py

```diff
@@ -3498,7 +3501,11 @@ def get_custom_entries(self):
         from .suite_xml.features.mobile_ucr import ReportModuleSuiteHelper
         return ReportModuleSuiteHelper(self).get_custom_entries()
 
-    def get_menus(self):
+    def get_menus(self, supports_module_filter=False):
+        kwargs = {}
+        if supports_module_filter:
+            kwargs['relevant'] = interpolate_xpath(self.module_filter)
```

Comment:
Module filtering is on feature preview, so I could imagine a user adding module filters, then turning the preview off, and expecting the filtering to be removed.